### PR TITLE
Feature/fetch api

### PR DESCRIPTION
**Pull request by predrag-babic**
fetch API

### CODE REVIEW

File: src/Components/Color/ContrastChecker.jsx [hunk 1]
```diff
@@ -5,20 +5,23 @@ export default function ContrastChecker({ hex, contrastText }) {
...
+          body: JSON.stringify({ colors: [contrastText, hex] }),
+          headers: {
+            "Content-Type": "application/json",
+          },
+        }
+      );
+      const contrastData = await response.json();
+      const contrastScore = contrastData.overall;
+
+      setResult(contrastScore);
```
Comment (Leon910) [inline src/Components/Color/ContrastChecker.jsx:21]: you could think of including a response error validation like `if (!response.ok) {
        throw new Error("Failed to fetch data!");` or try and catch

File: src/Components/Color/ColorCard.jsx [hunk 2]
```diff
@@ -54,6 +55,7 @@ export default function ColorCard({ color, onDeleteColor, onUpdateColor }) {
           </div>
           <h4>{color.role}</h4>
           <p>contrast Text: {color.contrastText}</p>
+          <ContrastChecker hex={color.hex} contrastText={color.contrastText} />
```
Comment (Leon910) [inline src/Components/Color/ColorCard.jsx:58]: I like that you created a new component for the contrast checking!

File: src/Components/Color/ContrastChecker.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        foregroundColor: contrastText,
+        backgroundColor: hex,
+      }),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    }
+  );
+
+  if (response.ok) {
```
Comment (Leon910) [inline src/Components/Color/ContrastChecker.jsx]: ahh sorry, here you have you error validation! perfect

**Review by Leon910 — COMMENTED** [review]
edit review

File: src/Components/Color/ContrastChecker.jsx (hunk 3 above)
Comment (predrag-babic) [reply src/Components/Color/ContrastChecker.jsx]: no it is outdated, but i know what you mean. I can include try and catch. 
thanks for the hint.

**Pull request merged**